src/utils/configuration.ts

```diff
@@ -33,19 +33,39 @@ const getServerConfigurationValue = <T>(configName: keyof typeof configurationSc
 }
 
 const getPortNumber = (): number => {
-    return getServerConfigurationValue<number>('port', 8084)
+    return getServerConfigurationValue<number>('port', 7863)
 }
 
-const getPath = (): string => {
-    return getServerConfigurationValue<string>('binaryPath', '.bin/runme')
+const getPath = (basePath: string): string => {
+    const baseUri = Uri.parse(basePath)
+    const binaryPath = getServerConfigurationValue<string>('binaryPath', 'bin')
+
+    // relative path
```

Comment:
Would the user ever realistically want a relative path from the extension folder? I feel like 99% of the time, if a user provides a `binaryPath`, it's gonna be an absolute path or a workspace path. I don't think there's any expectation that the user has any understanding of what's going on in the extension folder.

Probably the user's expectation is relative path -> relative to workspace, absolute -> absolute, not set -> bundled binary.

To make this work, we could pass the default binary path URI down, and then use `workspace.workspaceFolders[0]` if the provided path is relative, and fallback to the bundled binary if something goes wrong.

Reply:
Probably this option should not be configurable, and remove it.